src/core/isr.c

```diff
@@ -255,16 +255,33 @@ void page_fault_handler(stack_t* stack)
   // This handler needs the `error_code`, so let's retrieve it.
   uint64_t error_code = ((uint64_t*)stack)[-1];
 
+  int isPresent = (error_code >> 0) & 1;
```

Comment:
nit: we should probably use `uint8_t` here and we should use snake case, not camel case.

Reply:
Fixed